python/lsst/sims/movingObjects/linearObs.py

```diff
@@ -0,0 +1,293 @@
+from __future__ import print_function, division
+import os
+import warnings
+import numpy as np
+
+from itertools import repeat
+from scipy import interpolate
+
+import lsst.sims.photUtils.Bandpass as Bandpass
+import lsst.sims.photUtils.Sed as Sed
+
+from lsst.sims.utils import haversine
+from lsst.sims.utils import ModifiedJulianDate
+from lsst.sims.utils import ObservationMetaData
+from lsst.obs.lsstSim import LsstSimMapper
+from lsst.sims.coordUtils import chipNameFromRaDec
+
+from .orbits import Orbits
+from .ephemerides import PyOrbEphemerides
+
+__all__ = ['LinearObs', 'runLinearObs']
+
+
+class LinearObs(Orbits):
+    """
+    Class to generate observations of a set of moving objects.
+    Linear interpolation between gridpoint of ephemerides.
+    Inherits from Orbits to read orbits.
+    """
+    def __init__(self, orbitFile, delim=None, skiprows=None, ephfile=None,
+                 timescale='TAI', obscode=807):
+        super(LinearObs, self).__init__()
+        self.readOrbits(orbitfile=orbitFile, delim=None, skiprows=None)
+        self.pyephem = PyOrbEphemerides(ephfile=ephfile)
+        self.timescale = self.pyephem.timeScales[timescale]
```

Comment:
I guess I kind of figured that linearObs was more of a utility .. but sure, pretty cheap to add this.